examples/gitlab/postgres.yml

```diff
@@ -0,0 +1,22 @@
+name: postgresql
+containers:
+  - image: sameersbn/postgresql:9.5-3
+    volumeMounts:
+    - name: db
+      mountPath: /var/lib/postgresql
+    env:
+    - name: DB_USER
+      value: gitlab
+    - name: DB_PASS
+      value: +BP52QIxpT/flVCMpL3KXA==
+    - name: DB_NAME
+      value: gitlab_production
+    - name: DB_EXTENSION
+      value: pg_trgm
+services:
+  - name: postgresql
+    ports:
+    - port: 5432
+volumes:
+- name: db
+  emptyDir: {}
```

Comment:
This will lose data if the container is moved to another node.
`emptyDir` dir shouldn't be used here.